docs/library/pyb.DAC.rst

```diff
@@ -60,12 +60,27 @@ Constructors
    The maximum value for the write and write_timed methods will be
    2\*\*``bits``-1.
 
+   The *buffering* parameter selects the behaviour of the DAC op-amp output
+   buffer, whose purpose is to reduce the output impedance.  It can be
+   ``None`` to select the default (buffering enabled for :meth:`DAC.noise`,
+   :meth:`DAC.triangle` and :meth:`DAC.write_timed`, and disabled for
+   :meth:`DAC.write`), ``False`` to disable buffering completely, or ``True``
+   to enable output buffering.
+
+   When buffering is enabled the DAC pin can drive loads down to 5KΩ.
+   Otherwise it has an output impedance of 15KΩ maximum: consequently
+   to achieve a 1% accuracy the applied load should not exceed 1.5MΩ. Using
```

Comment:
@peterhinch I copied this from your PR (thanks!).  But I think it should say "should not be less than 1.5Mohm", don't you think?

Reply:
Of course. Sorry!